python/pyiceberg/table/__init__.py

```diff
@@ -259,7 +262,7 @@ def projection(self) -> Schema:
         return snapshot_schema.select(*self.selected_fields, case_sensitive=self.case_sensitive)
 
     @abstractmethod
-    def plan_files(self) -> Iterator[ScanTask]:
+    def plan_files(self) -> Iterable[ScanTask]:
```

Comment:
Is this change safe, or will it require updates to people that already call this?